snuba/query/allocation_policies/errors_allocation_policy.py

```diff
@@ -57,15 +58,25 @@ class ErrorsAllocationPolicy(AllocationPolicy):
 
     def __init__(
         self,
+        name: str,
         storage_set_key: StorageSetKey,
         required_tenant_types: list[str],
         **kwargs: str,
     ) -> None:
-        super().__init__(storage_set_key, required_tenant_types)
-
-    @property
-    def rate_limit_prefix(self) -> str:
-        return self.__class__.__name__
+        super().__init__(name, storage_set_key, required_tenant_types)
+        self._config_params.update(
+            {
+                "throttled_thread_number": AllocationPolicyConfig(
+                    "Number of threads any throttled query gets assigned.", int, 1
+                ),
+                "org_limit_bytes_scanned": AllocationPolicyConfig(
+                    "Number of bytes any org can scan in a 10 minute window.",
+                    int,
+                    10000,
+                ),
+            }
+        )
+        self._name = self._name or self.__class__.__name__
```

Comment:
This interface is a little confusing. You're updating state on the class in the init method rather than declaring what the configuration options are. There are two classes of configs:

1. Configs that apply to any policy (is_enforced, is_active)
2. Configs that apply to a specific policy

It shouldn't be the user's responsibility to correctly merge them, only to define them